config/manager/manager.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app.kubernetes.io/name: siteconfig-controller
+    app.kubernetes.io/component: siteconfig
+    control-plane: controller-manager
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/warn: privileged
+  annotations:
+    workload.openshift.io/allowed: management
+  name: system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: system
+  labels:
+    app.kubernetes.io/name: siteconfig-controller
+    app.kubernetes.io/component: siteconfig
+    control-plane: controller-manager
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: siteconfig-controller
+      app.kubernetes.io/component: siteconfig
+      control-plane: controller-manager
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: siteconfig-controller
+        app.kubernetes.io/component: siteconfig
+        control-plane: controller-manager
+      annotations:
+        openshift.io/scc: privileged
```

Comment:
Does this pod really need to be privileged?

Reply:
Changed to `openshift.io/scc: restricted`